### PR TITLE
Add initialize-using-composite-literal tenet

### PR DESCRIPTION
**Pull request by leilaes**

### CODE REVIEW

File: tenets/codelingo/effective-go/loop-variable-used-in-go-routine/test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			if n {
+				fmt.Println(val)
+			}
+		}(val, n)
+	}
+}
+
+func Test6(values []int, n bool) {
+	for val := range values {
+		go func(n bool) { //ISSUE
```
Comment (Daanikus) [inline tenets/codelingo/effective-go/loop-variable-used-in-go-routine/test.go]: What's wrong with this one?

File: tenets/codelingo/effective-go/loop-variable-used-in-go-routine/codelingo.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
+funcs:
+  - name: isEqualtoAny
+    type: asserter
+    body: |
+      function(name1, name2, name3) {
+        return (name1 == name3 && name1 != "_" ) || (name2 == name3 && name2 != "_" ) }
```
Comment (Daanikus) [inline tenets/codelingo/effective-go/loop-variable-used-in-go-routine/codelingo.yaml]: Move `}` to new line

File: tenets/codelingo/effective-go/loop-variable-used-in-go-routine/codelingo.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  - name: isEqual
+    type: asserter
+    body: |
+      function(name1, name2) {
+        return name1 == name2 && name1 != "_"  }
```
Comment (Daanikus) [inline tenets/codelingo/effective-go/loop-variable-used-in-go-routine/codelingo.yaml]: Move `}` to new line

File: tenets/codelingo/effective-go/loop-variable-used-in-go-routine/test.go (hunk 1 above)
Comment (leilaes) [reply tenets/codelingo/effective-go/loop-variable-used-in-go-routine/test.go]: Because `val` is used in the goroutine and not passed.

Comment (Daanikus) [reply tenets/codelingo/effective-go/loop-variable-used-in-go-routine/test.go]: Ah right. I thought the comment was saying the issue was with the bool.

**Pull request merged**